src/test/java/com/faforever/client/login/LoginControllerTest.java

```diff
@@ -17,15 +19,21 @@
 import static org.mockito.Mockito.when;
 
 public class LoginControllerTest extends AbstractPlainJavaFxTest {
+  @Value("${login.forgotLoginUrl}")
+  String forgotLoginUrl;
+  @Value("${login.createAccountUrl}")
+  String createUrl;
```

Comment:
`@Value` still doesn't work here, and it should still be a `private static final CREATE_ACCOUNT_URL = "http://www.example.com/create` :-)